src/mechanics/timeIntegration/TimeIntegrationBase.h

```diff
@@ -309,21 +256,21 @@ class TimeIntegrationBase
 	NuTo::StructureOutputBlockVector mLoadVectorTimeDependent;
 
 	//accumulated time (in case several loadings are looked at, one after another)
-	double mTime;
+	double mTime = 0.;
     //adapt the time step based on the number of iterations required (or decrease, if no convergence can be achieved)
-	bool mAutomaticTimeStepping;
+	bool mAutomaticTimeStepping = false;
 	//initial time step (or at the end this is the last time step used)
-	double mTimeStep;
-    //maximum time step (for adaptive simulations)
-	double mMaxTimeStep;
-    //minimum time step (for adaptive simulations)
-	double mMinTimeStep;
```

Comment:
Instead of using normal `//` comments on the line above, these should be inline doxygen comments `double mTime = 0.; //!< accumulated time`. I might add a commit to that effect later.